landoui/errorhandlers.py

```diff
@@ -94,7 +94,7 @@ def unexpected_error(e):
             title="Oops! Something went wrong.",
             message=(
                 "Something just went wrong. Try again or tell the team at "
-                "#lando on irc."
+                "#conduit on Matrix (https://chat.mozilla.org/#/room/#conduit:mozilla.org)."
```

Comment:
```suggestion
                "#conduit on Matrix (https://matrix.to/#/#conduit:mozilla.org)."
```

Forgot to check line length in my suggestion; also find a shorter link! Will merge after tests pass.